data_utils.py

```diff
@@ -32,20 +34,20 @@ def read_data(self):
             else:
                 self.data = json.load(f)
 
-        self.label_set = set(chain.from_iterable(row["output"] for row in self.data))
+        self.label_set = set(chain.from_iterable(row[self.output_key] for row in self.data))
 
     def dedupe_data(self, tokenizer):
         for i, line in enumerate(self.data):
             new_output = []
             tokenized_labels = []
-            for label in line['output']:
+            for label in line[self.output_key]:
                 tokenized = tuple(tokenizer(label).input_ids[:-1])
                 if tokenized not in tokenized_labels:
                     new_output.append(label)
                     tokenized_labels.append(tokenized)
                 else:
-                    print(f"Line {line['uid']} has repeated labels!")
-            line['output'] = new_output
+                    print(f"Line {line['id']} has repeated labels!")
```

Comment:
will this be an issue for other GROOV datasets where identifier is stored as 'uid' ?